src/components/atoms/Alert/Alert.tsx

```diff
@@ -24,29 +25,33 @@ export interface AlertProps extends React.HTMLAttributes<HTMLDivElement> {
 const Wrapper = styled.div<{ severity: Severity; inline: boolean; hasRoundedCorners: boolean }>`
   display: ${({ inline }) => (inline ? 'inline-flex' : 'flex')};
   position: relative;
-  padding: 8px 12px 8px 12px;
+  align-items: ${({ severity, theme }) => theme.alert[severity].alignItems};
+  padding: ${({ severity, theme }) => theme.alert[severity].padding ?? '8px 12px 8px 12px'};
   background: ${({ severity, theme }) => theme.alert[severity].background};
   color: ${({ severity, theme }) => theme.alert[severity].text};
-  font-size: ${typography.sizes.text.body};
-  line-height: ${typography.sizes.lineHeight.body};
-  font-family: ${typography.primary};
-  font-weight: 400;
-  border-radius: ${({ hasRoundedCorners }) => (hasRoundedCorners ? 4 : 0)}px;
+  font-size: ${({ theme }) => theme.typography.text.sizes.body ?? typography.sizes.text.body};
+  line-height: ${({ theme }) => (theme.typography.lineHeight.small ? typography.sizes.lineHeight.body : '24px')};
+  font-family: ${({ theme }) => theme.typography.primary ?? typography.primary};
+  font-weight: ${({ theme }) => theme.typography.weights.ultraBold ?? '400'};
+  border-radius: ${({ severity, theme, hasRoundedCorners }) =>
+    theme.alert[severity].borderRadius ? theme.alert[severity].borderRadius : hasRoundedCorners ? '4px' : '0px'};
   border: ${({ severity, theme }) => theme.alert[severity].border};
+  border-left: ${({ severity, theme }) => theme.alert[severity].borderLeft};
+  width: ${({ severity, theme }) => theme.alert[severity].width};
+  height: ${({ severity, theme }) => theme.alert[severity].height};
   a {
     color: ${({ severity, theme }) => theme.alert[severity].text} !important;
     font-size: 16px;
     line-height: 20px;
     text-decoration: underline;
-
     &:hover {
       text-decoration: none;
     }
   }
 `;
 
 const IconWrapper = styled.div<{ severity: Severity }>`
-  margin-right: 8px;
+  margin-right: ${({ severity, theme }) => theme.alert[severity].marginRight ?? '8px'};
```

Comment:
use variable from sizes instead of hardcoded 8px

Reply:
suppose that we shouldn’t be using hardcoded value, but what would you do with the value was hardcoded before?

Reply:
if you spot hardcoded value but you can use the variable, use variable :)

Reply:
some custom values can still be hardcoded, e.g. 22.5px, we don't have variable for that. 8px is `${spacing[2]}`

Reply:
cool. so you can see the comments here as well. I thought I needed to push it first. That's cool. Loving this!